ts/Series/Lollipop/LollipopPoint.ts

```diff
@@ -63,7 +71,7 @@ extend(LollipopPoint.prototype, {
         series: LollipopSeries,
         options: LollipopPointOptions,
         x?: number
-    ): Point {
+    ): typeof Point.prototype {
```

Comment:
In case this is the only usage of the Point class in the file, it is better to not use deconstruction, but an `import type`. Sorry for the confusion.

Reply:
This function also calls `Point.prototype.init` :)

Reply:
Yes, okay. Well then everything is settled. 😅